### PR TITLE
DOCS-858: Add crop to transform camera

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/camera/transform.md [hunk 1]
```diff
@@ -96,6 +96,32 @@ Classifications overlay text from the `GetClassifications` method of the [Vision
...
+}
+
+```
+
+**Attributes:**
+
+- `x_min_px`: The x coordinate of the first point that specifies the rectangular area to crop the image to.
+- `y_min_px`: The y coordinate of the first point that specifies the rectangular area to crop the image to.
+- `x_max_px`: The x coordinate of the second point that specifies the rectangular area to crop the image to.
+- `y_max_px`: The y coordinate of the second point that specifies the rectangular area to crop the image to.
```
Comment (bhaney) [inline docs/components/camera/transform.md:121]: [optional] If it's clearer, the first point is the "top left" point of the cropping rectangle, and the second point would be the "bottom right" point of the cropping rectangle.

**Review by bhaney — APPROVED** [review]
Nice, thanks for this!

File: docs/components/camera/transform.md (hunk 1 above)
Comment (npentrel) [reply docs/components/camera/transform.md:121]: Thank you - I was gonna ask that. I tried to test it and then realized it's not merged yet - so I'll chagne that and then just let me know when it's good to go :)

Comment (npentrel) [inline docs/components/camera/transform.md:121]:
```suggestion
- `x_min_px`: The x coordinate of the top left point of the rectangular area to crop the image to.
- `y_min_px`: The y coordinate of the top left point of the rectangular area to crop the image to.
- `x_max_px`: The x coordinate of the bottom right point of the rectangular area to crop the image to.
- `y_max_px`: The y coordinate of the bottom right point of the rectangular area to crop the image to.
```

**Pull request merged**